app/styles/newcomponents/ilios-chart-tooltip.scss

```diff
@@ -1,37 +0,0 @@
-.ilios-chart-tooltip {
```

Comment:
Good catch, I've pushed a new version of simple-charts and required it in this branch.